src/wp-includes/class-wp-term.php

```diff
@@ -103,17 +103,19 @@ final class WP_Term {
 	 * Retrieve WP_Term instance.
 	 *
 	 * @since 4.4.0
+	 * @since 6.4.0 Added the `$filter` parameter.
 	 *
 	 * @global wpdb $wpdb WordPress database abstraction object.
 	 *
 	 * @param int    $term_id  Term ID.
 	 * @param string $taxonomy Optional. Limit matched terms to those matching `$taxonomy`. Only used for
 	 *                         disambiguating potentially shared terms.
+	 * @param string|false $filter   Optional. How to sanitize term fields. Default 'raw'.
```

Comment:
```suggestion
	 * @param int          $term_id  Term ID.
	 * @param string       $taxonomy Optional. Limit matched terms to those matching `$taxonomy`. Only used for
	 *                               disambiguating potentially shared terms.
	 * @param string|false $filter Optional. How to sanitize term fields. Default 'raw'.
```

Correct indention for `param`